### ISSUE TITLE
Downloading issues.

### ISSUE THREAD

**Issue by gforhan** [eshapard/AnkiHabitica#14]
Hello! I'm really excited about using this addon and think it is a fantastic idea!
However, using the download code that is currently on the Anki website I have run into a problem.
Everything seems to download just fine and there are no error messages. However, when I tell Anki my user ID and API number, it says that it was successful but then nothing happens. 

I am AWFUL at most things computer so I am having a hard time figuring out how to fix this. Any help would be greatly appreciated! :)

**Comment by eshapard** [comment]
Hmm... would you post the contents of the config file? It'll be in the
addons folder where your Anki profile is (in windows, that's your Documents
folder). The config file is .ankihrpg.conf for the version you are using.

We want to make sure that your userid and apitoken are stored correctly in
the config file... if there's a problem with them, the addon might not bug
you about it.

You can test that your userid and apitoken are correct by plugging them in
to: https://habitica.com/static/api-v2 and seeing if you can export your
user history (hit the Try it Out! button).

Because your config file has the apitoken in it, and that can be used to
log in to your account, just replace some of the digits with big Xs; that
way it'll be safe to post it on github. (posting your user-id is fine).

I'm working on a new version 2.0 that I hope to release at the end of June.
Come July, habitica will take down version 2 of the API and the previous
versions of ankiHRPG will stop working, so I'm under some pressure to get
this new version ready by then.

**Comment by eshapard** [comment]
Oh, and the version you have may not show anything until you've scored enough points. (It's been a while since I worked on that version). Have you gone through lots of cards with it installed?

**Comment by gforhan** [comment]
Thank you for your very detailed and quick response!
Last night I kept playing with it but couldn't get anything to work. I came across a similar website to the one you posted that gets all the info from my account and displays it for me when I give it my user ID and API code, so I was definitely using the right numbers. 

However, I eventually downloaded the files you have here on github and manually copied and pasted the contents of the AnkiHabitica_v2.0 folder  (So I had ankihabitica.py and the AnkiHabitica folder) into my addons folder on anki. Doing this seems to have made everything run perfectly. However, I no longer have any of of the old files from attempting to use the download code on the anki website. 

If it could be of any use to you in making your program better I'd be more than willing to delete my setup now and try to install using the download code again! However, while I have your attention, I was wondering if you could please quickly explain what the "Score Habitica Backlog" button does?

**Comment by eshapard** [comment]
I'll be replacing the old one on ankiweb soon anyway, so let's not bother.

Score Habitica Backlog scores any backlogged points you've racked up that
the program hasn't scored you for on habitica.

For example, if you did a bunch of reviews on a mobile device, or if you
did a bunch of reviews while not connected to the internet, then the addon
is going to see a bunch of un-scored points that you've earned. It'll work
those extra points in here and there as you use Anki, or you can choose to
take care of them all at once.

Basically, when you have a backlog of points, it will score you up to three
times everytime your progress bar reaches the end to catch up.

**Comment by eshapard** [comment]
The official version 2.0 of the Anki Habitica add-on was just published. Please upgrade :-) 

https://ankiweb.net/shared/info/954979168
